reference/forms/types/options/choice_value.rst.inc

```diff
@@ -4,20 +4,55 @@ choice_value
 .. versionadded:: 2.7
     The ``choice_value`` option was introduced in Symfony 2.7.
 
-**type**: ``callable`` or ``string`` **default**: ``null``
+**type**: ``callable``, ``string`` or :class:``Symfony\\Component\\PropertyAccess\\PropertyPath`` **default**: ``null``
 
 Returns the string "value" for each choice. This is used in the ``value`` attribute
 in HTML and submitted in the POST/PUT requests. You don't normally need to worry
 about this, but it might be handy when processing an API request (since you can
 configure the value that will be sent in the API request).
 
-This can be a callable or a property path. See `choice_label`_ for similar usage.
-If ``null`` is used, an incrementing integer is used as the name.
+This can be a callable or a property path. Both have the responsibility to return
+a unique string for each choice, otherwise an incrementing integer is used.
+
+If it is a callable, it get passed each choice as only argument, supposed your choices
+are some previously loaded entities in a given array::
+
+    $builder->add('attending', 'choice', array(
+        'choices' => $entities,
+        'choices_as_values' => true,
+        'choice_value' => 'id',
+    ));
+
+Is equivalent to::
+
+    $builder->add('attending', 'choice', array(
+        'choices' => $entities,
+        'choices_as_values' => true,
+        'choice_value' => function ($choice) {
+            return $choice->getId();
+        },
+    ));
+
+.. tip::
+
+    Those examples reproduce the default behavior of the ``EntityType``.
+    When your choices are already loaded you should consider using the
+    ``ChoiceType`` instead unless you need a lazy loading or a custom query.
```

Comment:
I'm not sure what this is adding - especially the second sentence - it sounds like I should prefer to use the ChoiceType when I need a drop-down of entities.